{"name": "Test Fail Get User Searches - Auth Denied", "status": "passed", "start": 1739387121370, "stop": 1739387121375, "uuid": "ed46b60b-1fd8-4161-834a-6f56971eb809", "historyId": "ccb4f5cad128b3b4545dfdf5b506e0b3", "testCaseId": "ccb4f5cad128b3b4545dfdf5b506e0b3", "fullName": "tests.test_api.test_users#test_get_searches_fail_server_error", "labels": [{"name": "parentSuite", "value": "tests.test_api"}, {"name": "suite", "value": "test_users"}, {"name": "host", "value": "YRLenovo"}, {"name": "thread", "value": "20776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api.test_users"}]}